install/source.rst

```diff
@@ -227,6 +227,8 @@ Step 4: Initialize your database
 
    Ensure to do this as ``zammad`` user in your Zammad directory!
 
+.. include:: /install/includes/db-seed-base_url.rst
+
```

Comment:
If there is only a single inclusion of this file, I'd like to argue we don't really need this as a separate file. Modularity is not really the goal here, we leverage `include` to avoid repetition only.